Sources/SolToBoogie/ProcedureTranslator.cs

```diff
@@ -2445,11 +2445,28 @@ private void TranslateCallStatement(FunctionCall node, List<BoogieIdentifierExpr
         private void TranslateTransferCallStmt(FunctionCall node)
         {
             var amountExpr = TranslateExpr(node.Arguments[0]);
+            var memberAccess = node.Expression as MemberAccess;
+            var baseExpr = memberAccess.Expression;
+            var retVar = MkNewLocalVariableWithType(BoogieType.Bool);
 
-            // call FallbackDispatch(from, to, amount)
-            BoogieCallCmd callStmt = MkFallbackDispatchCallCmd(node, amountExpr);

```

Comment:
Why are you removing this call? This changes the default behavior even when your revert logic is not triggered.

Reply:
As I mentioned in the pull request, I changed the modeling of transfers to require(send(from, to, amount));

The fallback is still being called through the call to send. I needed a flag that indicates whether the transfer failed or not. Otherwise, I would have to implement the revert logic within the dispatch method which felt a bit unnatural.